src/MetadataUtility/Audio/Wave.cs

```diff
@@ -353,7 +353,16 @@ public static Fin<RangeHelper.Range> ScanForChunk(Stream stream, RangeHelper.Ran
                 // check the chunk length falls within the bounds of the file
                 if (offset + length > stream.Length)
                 {
-                    return InvalidChunk;
+                    // FL005 is detected - decrement length by 44 for an accurate value
+                    // TODO: Fix this problem rather than cover it up
+                    if (offset + length - stream.Length == 44)
+                    {
+                        length -= 44;
+                    }
+                    else
+                    {
+                        return InvalidChunk;
+                    }
```

Comment:
I'd really like the FL005 patch factored out into it's own method - otherwise it's buried in an very obscure place

Reply:
Sounds good... also moved 44 out into a constant